src/test/java/com/mercadopago/client/order/OrderClientTest.java

```diff
@@ -202,6 +203,41 @@ void captureSuccess() throws MPException, MPApiException, IOException {
         Assertions.assertEquals(order.getStatus(), "processed");
     }
 
+    @Test
+    void refundTotalSuccess() throws MPException, MPApiException, IOException {
+        HttpResponse response = MockHelper.generateHttpResponseFromFile(CREATE_REFUND_TOTAL_RESPONSE_FILE, HttpStatus.OK);
+        Mockito.doReturn(response).when(HTTP_CLIENT).execute(any(HttpRequestBase.class), any(HttpContext.class));
+
+        String id = "01JCK2RRKV10XVTEBJR598QH9Z";
+
+        OrderRefund orderRefund = client.refundTotal(id);
+
+        Assertions.assertNotNull(orderRefund);
+        Assertions.assertEquals(HttpStatus.OK, orderRefund.getResponse().getStatusCode());
```

Comment:
Adicionar testes validando se os atributos retornados estão iguais ao mockado, isso garante que estamos respeitando o body.
```suggestion
        Assertions.assertNotNull(orderRefund);
        Assertions.assertEquals(HttpStatus.OK, orderRefund.getResponse().getStatusCode());
        assertNotNull(orderRefund.getResponse());
        Assertions.assertEquals("refunded", orderRefund.getStatus());
        Assertions.assertEquals("ref_01JCK7R5R73JCYXRCKFH7MJR9K", orderRefund.getTransactions().getRefunds().get(0).getId());
```